### ISSUE TITLE
Choose graphics card to use in VSCode

### ISSUE THREAD

**Issue by AlenPelin** [microsoft/vscode#75698]
**TL;DR:** how to choose which graphics card to use in VSCode? if it is not possible, please consider it as a wish.

I am performance issues with **VSCode 1.34.0** running on Windows 10 system with **two GPU** used at the same time to power two 4k monitors: 
* Vega 11 which is part of Ryzen 2400g, and 
* GeForce 1050Ti

I noticed that after reboot, after some time of intense work with the system, TaskManager shows constant 100% GPU memory utilization (2GB of total GB) for Vega and just about 40% of GeForce (of total 4GB) which instantly drops after closing VSCode:

![Clipboarder 2019 06 18](https://user-images.githubusercontent.com/2854666/59689607-48899e80-9223-11e9-9a82-d1e0c68a1616.png)

The search in the Net didn't give any results on how to specify which GPU unit to use when running VSCode because it is really pathetic to disable GPU completely when having good graphics card available and waiting to do some really good work.

**Comment by AlenPelin** [comment]
Okay, it looks like it can be changed by NVidia Control Panel. I'll give it a try and update this one.

**Comment by sandy081** [comment]
Please update us if it helped you or not.

**Comment by AlenPelin** [comment]
Well, it noticeably helped. Now it uses memory of both GPUs, but Radeon is used minorly. 

I'll leave it here for others:
![image](https://user-images.githubusercontent.com/2854666/59747738-d9608880-92bc-11e9-887e-8cbaaef40adf.png)
